pkgs/applications/misc/bashSnippets/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchFromGitHub
+, curl, netcat, mpv, python, bind, iproute, bc, gitMinimal, ... }:
+let
+  version = "1.12.0";
+in
+stdenv.mkDerivation {
+  name = "bashSnippets-${version}";
+
+  src = fetchFromGitHub {
+    owner = "alexanderepstein";
+    repo = "Bash-Snippets";
+    rev = "v${version}";
+    sha256 = "0kx2a8z3jbmmardw9z8fpghbw5mrbz4knb3wdihq35iarcbrddrg";
+  };
+
+  propagatedBuildInputs = [
```

Comment:
Substituting the paths in the source files. Input propagation has to do with the build environment, it's not intended as a mechanism for augmenting the runtime environment.